spec/controllers/archive_faqs_controller_spec.rb

```diff
@@ -6,38 +6,43 @@
   include LoginMacros
   include RedirectExpectationHelper
 
-  describe "GET #index" do
-    it "renders the index page anyway when the locale param is invalid" do
-      expect(I18n).to receive(:with_locale).with("eldritch").and_call_original
-      get :index, params: { language_id: "eldritch" }
-      expect(response).to render_template(:index)
-    end
+  let(:non_standard_locale) { create(:locale) }
+  let(:user_locale) { create(:locale) }
+  let(:user) do
+    user = create(:user)
+    user.preference.update!(preferred_locale: user_locale.id)
+    $rollout.activate_user(:set_locale_preference, user)
```

Comment:
I'm not sure if this is on for everyone in prod, but it might be nice to test when it's disabled as well